src/nvim/regexp_nfa.c

```diff
@@ -6064,10 +6070,17 @@ static int nfa_regmatch(nfa_regprog_T *prog, nfa_state_T *start, regsubs_T *subm
       break;
 
     // Allow interrupting with CTRL-C.
-    fast_breakcheck();
+    line_breakcheck();
```

Comment:
@oni-link How do you reproduce this? I've tried running `make test` on three different computers and everything works correctly.